ecs-cli/modules/cli/regcreds/regcreds_app.go

```diff
@@ -278,3 +287,10 @@ func getNewCommandConfig(c *cli.Context) *config.CommandConfig {
 
 	return commandConfig
 }
+
+func validateOutputOptions(outputDir string, skipOutput bool) error {
+	if outputDir != "" && skipOutput {
+		return fmt.Errorf("both output directory ('%s') and '--"+flags.NoOutputFileFlag+"' specified", outputDir)
```

Comment:
Can this be rephrased to be more explicit? e.g. "Only one of x and y can be specified but both flags were present"